app/assets/javascripts/components/timeline/timeline.jsx

```diff
@@ -64,6 +64,13 @@ const Timeline = createReactClass({
     return week.blocks;
   },
 
+  submit(e) {
```

Comment:
I don't understand what's going on with all the changes to this file. It doesn't look like this `submit` function is used anywhere.

Can you explain more fully what the strategy of this change is, and clean up any changes to this file that aren't necessary?

Reply:
@ragesoss sorry my bad. There's some unwanted code in the file. I'll remove that and make a commit

Reply:
And about the other changes, basically what I've done is I check whether there is a banner or not and do the styling accordingly. I got the if conditions from course_alerts.jsx

Reply:
Thanks for the explanation! That helps me understand it.

Reply:
Is what I have done fine?